doc/source/whatsnew/v0.19.0.txt

```diff
@@ -222,6 +231,46 @@ New behaviour:
 
    In [2]: pd.read_csv(StringIO(data), names=names)
 
+
+.. _whatsnew_0190.enhancements.read_csv_categorical:
+
+:func:`read_csv` supports parsing ``Categorical`` directly
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
the underline needs to be at least as long as the title (otherwise you get warnings in the doc build, not that bad, but like to keep the number of warnings as low as possible so the important warnings do not get lost)